app/src/main/java/org/schabi/newpipe/RouterActivity.java

```diff
@@ -43,6 +44,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     protected void handleUrl(String url) {
+        StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder().permitAll().build()); // FIXME
         boolean success = NavigationHelper.openByLink(this, url);
```

Comment:
I can not merge. The function `NaviagationHelper.openByLink()` seems not to bee exising in current dev branch.
@wb9688 could you please fix this merge collison.